sofarpc-scenario/config/docker-composexx.yml

```diff
@@ -0,0 +1,57 @@
+version: '2.1'
+services:
+  zk-server:
+    image: zookeeper:3.4.10
+    expose:
+      - "2181"
+    ports:
+      - "2181:2181"
+
+  sofarpc-consumer:
+    image: skywalking/sofarpc-consumer-scenario:5.3.2
```

Comment:
The image version should be `{CASES_IMAGE_VERSION}` instead of an detail version